doozer

```diff
@@ -1454,6 +1454,36 @@ def config_print(runtime, key, name_only, as_yaml):
     config.config_print(key, name_only, as_yaml)
 
 
+@cli.command("config:gen-csv", short_help="Generate .csv file for given images/rpms")
+@click.option("--fields", help="Specific key in config to print, separated by commas: --fields key,name,owners",
```

Comment:
Now that I'm looking at this I feel like `--keys` would be more in line with other usages in doozer.